tests/ttnn/unit_tests/test_to_and_from_torch.py

```diff
@@ -84,3 +84,21 @@ def test_from_torch_large(device):
     x_tensor = ttnn.from_torch(torch_x, layout=ttnn.TILE_LAYOUT)
     x_tensor = ttnn.to_torch(x_tensor)
     assert torch.allclose(torch_x, x_tensor)
+
+
+@pytest.mark.parametrize(
+    "shape",
+    [
+        (),
+        (1),
+        (2),
+        (0),
+    ],
+)
+@pytest.mark.parametrize("layout", [ttnn.ROW_MAJOR_LAYOUT, ttnn.TILE_LAYOUT])
+@pytest.mark.parametrize("dtype", [torch.float32, torch.bfloat16])
+def test_to_for_01_rank(shape, layout, dtype):
+    torch_input_tensor = torch.rand(shape, dtype=dtype)
+    tensor = ttnn.from_torch(torch_input_tensor, layout=layout)
+    # torch_output_tensor = ttnn.to_torch(tensor)
```

Comment:
Why is the to torch and verification commented out?

Reply:
I think Stas accidentally made it a non-draft

Reply:
This is intentional. Conversion `to_torch` is not yet supported, I intend to fix it and uncomment the lines in the next PR. Added a comment about it.

Reply:
I am a little uneasy pushing something in without being able to test if it can cause data corruption (pcc issues). This test really only says that it doesn't assert out or reach an illegal case. The others can weigh in if they disagree. Maybe you could manually check if the tensor being produced does match expectations by printing it out if there is no alternative?

Reply:
What is existing behaviour vs new behaviour?

If the new behaviour is no worse than existing behaviour then it should definitely be ok.

If new behaviour is worse, would a warning be sufficient for now, until the next PR?

Reply:
I guess given the other tests work and we didn't previously support 0/1D this doesn't break anything that previously worked so it's ok but it definitely needs to be well tested in the next pr.

Reply:
@jvegaTT Yes, this test is not that great. I intend to make another PR which would allow to uncomment the data verification part of the test.
I strongly believe the PR needs to be merged as-is. The change is already pretty-large, and if I keep adding on things it'll become too large to review. Moreover it keeps accumulating merge conflicts quite rapidly.

@bbradelTT 
The existing behavior is TT_FATAL on `from_torch` call. The new behavior is that `from_torch` succeeds (with hopefully correct data), but subsequent calls like `to_torch` continue to fail with TT_FATAL for those kinds of tensors.

Reply:
I agree. I sent the approval on my end